### DISCUSSION TITLE
Integration to HuggingFace 🤗

### DISCUSSION THREAD

**Discussion by subercui** [bowang-lab/scGPT#115] [Ideas]
Discussion about how to integrate scGPT onto Hugging Face 🤗 , and for sharing the progress. Hosting the model on HF will expand the accessibility to users.

**Comment by katielink** [comment]
Here are some docs and tutorials to get started! 🤗

- Sharing models to the HF Hub: https://huggingface.co/docs/transformers/model_sharing
- Sharing a custom model in transformers: https://huggingface.co/docs/transformers/custom_models
- 1st part of a Youtube series on contributing a model to HF: https://www.youtube.com/watch?v=wElORCdXHTU

**Comment by maalbadri** [comment]
@subercui Is there an update on this? What is the state of this task? I can see the last commit on the integrate-huggingface-model branch was 5 months ago and AFAIK there's no trace on HF for the pretrained model. Cheers.

**Comment by Josephrp** [comment]
Hi there, i'm an amateur , but very keen to help complete the last taskbox in the task list (this task) , what do you think if i just make a demo and simply publish it as a proof of concept ?
